### DISCUSSION TITLE
Download & View

### DISCUSSION THREAD

**Discussion by hirockstar** [iRedScarf/onedrive-index#2] [Q&A]
Is there any way to add function that can show how many times file/folder view or download.

**Comment by iRedScarf** [comment]
> Is there any way to add function that can show how many times file/folder view or download. 

Using Vercel Analytics, you can see the number of times the page has been visited, and the number of downloads has not been recorded for the time being. If necessary, let's study together how to implement the counting function of being browsed and downloaded.

**Reply by hirockstar** [reply]
Well as of I read there api documentation there is no APIs that provide that. 
But what we can do is we can create function that store the value to database based on file id.

**Reply by iRedScarf** [reply]
This project is to deploy the front end on Vercel, and download files directly from OneDrive's server by calling the API, which is free of charge. It should be a feasible solution to create a function to record the number of page views and the number of clicks on file download links.

**Reply by hirockstar** [reply]
Yeah true
